packages/backend/src/database/entities/scheduler.ts

```diff
@@ -30,6 +30,7 @@ export type SchedulerDb = {
     thresholds: string | null;
     enabled: boolean;
     notification_frequency: string | null;
+    selected_tabs: string | null;
```

Comment:
this does not match with the jsonb type on db, specified on the migration. I'd use `string[] | null` here and `text[]` type on db